fastlane/Fastfile

```diff
@@ -63,77 +245,21 @@ platform :ios do
                 sh "cd .. && cp -R ~/tmp/cache/Carthage ./Carthage || echo ⚠️ Carthage cache not exist, skip copying"
```

Comment:
I think we can remove all this carthage cache no? since you add a different logic for it